### ISSUE TITLE
[Build] 

### ISSUE THREAD

**Issue by madurais** [microsoft/onnxruntime#13554]
### Describe the issue

Hi, 

In onnxruntime_test_utils library (libonnxruntime_test_utils.a) code 

extern std::unique_ptr<Ort::Env> ort_env;

is declared as extern. The binaries which uses the library is expected to have ort_env declared and initialized as in 
onnxruntime/test/onnx/main.cc, onnxruntime/test/global_thread_pools/test_main.cc. But in code of onnxruntime_perf_test and onnxruntime_api_tests_without_env, there is no declaration of ort_env, but onnxruntime_api_tests_without_env  includes onnxruntime_test_utils through onnxruntime_shared_lib_test_LIBS and  onnxruntime_perf_test  includes through onnxruntime_perf_test_libs in cmake/onnxruntime_unittests.cmake. 

This causes compilation failure in some environment like AIX as
ld: 0711-317 ERROR: Undefined symbol: ort_env

We added the code to declare and initialize the ort_env in onnxruntime/test/api_tests_without_env/test_apis_without_env.cc, and onnxruntime/test/perftest/main.cc as a workaround. 

Please suggest is there any other better way of having a common file for declaration and initialization of the ort_env variable. 

### Target platform

AIX

### Build script

build.sh

### Error / output

ld: 0711-317 ERROR: Undefined symbol: ort_env

**Comment by snnn** [comment]
If I understand correctly, the symbol is not actually used in these programs. So it is not an issue on Linux or Windows. But AIX warns it?

**Comment by madurais** [comment]
@snnn,  Yes. The symbol is used in the library but expected to declared in binary. Some binaries do not use the functions that uses the symbol. Yet AIX gives error for the symbol undefined, while Linux or Windows does not give error.

**Comment by madurais** [comment]
We found an option in AIX ld command - -Wl,-berok. This avoids error for unreferenced external symbols. Using this flag solves the issue.
